source/math/FloatMatrix.ooc

```diff
@@ -45,6 +45,36 @@ FloatMatrix : cover {
 		result
 	}
 
+	setColumn: func (x: Int, values: ...) {
```

Comment:
Like this:
```ooc
set(data: FloatMatrix2D, x, y: Int) {
// check x,y >= 0
// check x+data width < this width, and the same for height
for (i in 0 .. data width)
  for (j in 0 .. data height)
    this[x+i, y+j] = data[i,j]
}
```
And similar for FloatPoint2D/FloatPoint3D and so on. Setting a column or a row would mean inputting a `1xN` or a `Nx1` FloatMatrix.